### DISCUSSION TITLE
Protobuf

### DISCUSSION THREAD

**Discussion by ACrsm** [yimengfan/BDFramework-Unity3d#68] [Q&A] [answered]
奶大，请教几个问题：
1：框架中的google protobuf 使用的是什么版本，跟官方相比有什么优化改动嘛？
2： 我想移植到其他项目，是不是直接将 3rdPlugins下的Protobuf , 以及Protobuf2ClassTools 拷贝走就行了。

**Comment by li10400** [comment]
已收到，谢谢！

**Comment by yimengfan** [comment, marked as answer]
没什么优化，只对ILRuntime时候有部分修改，如果用hyclr的话应该无所谓，具体代码部分你根据 按钮功能相关代码 全部挪过去就好了
